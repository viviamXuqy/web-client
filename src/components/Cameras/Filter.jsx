import React, { PureComponent } from 'react';
import { object, func } from 'prop-types';
import { Form, Button, Input, Icon } from 'antd';
import { injectIntl } from 'react-intl';

import classNames from 'classnames';

import Cicon from '../../shared/components/Cicon';
import Cookies from 'js-cookie';

class Filter extends PureComponent {
  static propTypes = {
    intl: object.isRequired,
    form: object.isRequired,
    onSearch: func.isRequired,
    onBayonetAdd: func.isRequired,
    onBayonetNewAdd: func.isRequired,
    onBayonetNewRtspAdd: func.isRequired,
    onUploadFile: func.isRequired,
  }

  state = {
    isReset: false, // 是否显示重置刷新按钮
  }

  handleSubmit = e => {
    e.preventDefault();

    const { onSearch } = this.props;

    this.props.form.validateFieldsAndScroll((err, values) => {
      if (!err) {
        onSearch(values);
        this.setState({
          isReset: true,
        });
      }
    });
  }

  handleReset = () => {
    const { onSearch } = this.props;
    const options = { bayonetName: '' };
    onSearch(options);

    this.setState({
      isReset: false,
    });
  }

  handleUpload = () => {
  }

  render() {
    const {
      form, onBayonetAdd, onBayonetNewAdd, onBayonetNewRtspAdd, onUploadFile, intl,
    } = this.props;
    const { getFieldDecorator } = form;
    const { isReset } = this.state;
    const user = JSON.parse(Cookies.get('user'));
    return (
      <div className="cameras-search">
        <Form
          className={classNames('app-page__filter', 'clearfix')}
          layout="inline"
          onSubmit={this.handleSubmit}
        >
          <div className="fl">
            <Form.Item>
              {getFieldDecorator('bayonetName')(
                <Input placeholder={intl.formatMessage({ id: 'App.ipc.search.placeholder' })} />,
              )}
            </Form.Item>
            <Form.Item className="buttons-group">
              <Button type="primary" htmlType="submit" icon="search">
                {intl.formatMessage({ id: 'App.ipc.searchBtn' })}
              </Button>
            </Form.Item>
          </div>
          <div>
            <Form.Item label={`${intl.formatMessage({ id: 'App.ipc.addCamera' })}:`} className="buttons-group">
              <Button
                type="primary"
                onClick={onBayonetNewRtspAdd}
              >
                <Cicon type="addcamera" />{intl.formatMessage({ id: 'App.ipc.quickAdd.addcamera' })}
              </Button>
              <Button
                type="primary"
                onClick={onBayonetAdd}
                disabled={user.grade === '3'}
              >
                <Cicon type="alreadyhave" />{intl.formatMessage({ id: 'App.ipc.existCheckpoint' })}
              </Button>
              <Button
                type="primary"
                onClick={onBayonetNewAdd}
                disabled={user.grade === '3'}
              >
                <Cicon type="addcamera" />{intl.formatMessage({ id: 'App.ipc.addCheckpoint' })}
              </Button>
              <Button
                type="primary"
                onClick={onUploadFile}
                disabled={user.grade === '3'}
              >
                <Icon type="upload" />{intl.formatMessage({ id: 'App.ipc.upload' })}
              </Button>
              <Button
                style={{ display: (isReset ? '' : 'none') }}
                type="primary"
                onClick={this.handleReset}
              >
                <Cicon type="therefresh" />{intl.formatMessage({ id: 'App.control.reset' })}
              </Button>
            </Form.Item>
          </div>
        </Form>
      </div>
    );
  }
}

export default Form.create({
  mapPropsToFields({
    filter: {
      bayonetName = '',
    },
  }) {
    const { createFormField } = Form;

    return {
      bayonetName: createFormField({
        value: bayonetName,
      }),
    };
  },
})(injectIntl(Filter));
