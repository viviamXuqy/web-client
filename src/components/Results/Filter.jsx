import React, { PureComponent } from 'react';
import { object, func } from 'prop-types';
import { Form, Button, Input, DatePicker, message } from 'antd';
import { injectIntl } from 'react-intl';
import moment from 'moment';

class Filter extends PureComponent {
  static propTypes = {
    intl: object.isRequired,
    form: object.isRequired,
    onSearch: func.isRequired,
  }

  handleSubmit = e => {
    e.preventDefault();

    const { onSearch, intl } = this.props;

    this.props.form.validateFieldsAndScroll((err, {
      range,
      taskBeginId,
      taskEndId,
      ...values
    }) => {
      if (!err) {
        if ((taskBeginId && !taskEndId) || (taskEndId && !taskBeginId)) {
          message.warn(intl.formatMessage({ id: 'App.results.rules.taskId' }));
          return;
        }

        const [begin, end] = range;
        const options = {
          ...values,
          taskBeginId,
          taskEndId,
          begin: begin && new Date(begin).getTime(),
          end: end && new Date(end).getTime(),
        };

        onSearch(options);
      }
    });
  }

  render() {
    const { form, intl: { formatMessage } } = this.props;
    const { getFieldDecorator } = form;

    return (
      <div>
        <Form
          className="app-page__filter"
          layout="inline"
          onSubmit={this.handleSubmit}
        >
          <div className="fl">
            <Form.Item>
              {getFieldDecorator('search')(
                <Input className="result-input" placeholder={formatMessage({ id: 'App.form.rules.keyword' })} />,
              )}
            </Form.Item>
            <Form.Item>
              {getFieldDecorator('range')(
                <DatePicker.RangePicker
                  style={{ width: 330 }}
                  showTime={{
                defaultValue: [moment('00:00:00', 'HH:mm:ss'), moment('23:59:59', 'HH:mm:ss')],
              }}
                  format="YYYY-MM-DD HH:mm:ss"
                />,
              )}
            </Form.Item>
            <Form.Item>
              {getFieldDecorator('taskBeginId')(
                <Input className="result-input" placeholder={formatMessage({ id: 'App.results.beginTaskId' })} />,
              )}
            </Form.Item>
            <div className="results-list__lable">è‡³</div>
            <Form.Item>
              {getFieldDecorator('taskEndId')(
                <Input className="result-input" placeholder={formatMessage({ id: 'App.results.endTaskId' })} />,
              )}
            </Form.Item>
            <Form.Item className="buttons-group">
              <Button type="primary" htmlType="submit" icon="search">
                {formatMessage({ id: 'App.control.search' })}
              </Button>
            </Form.Item>
          </div>
        </Form>
      </div>
    );
  }
}

export default Form.create({
  mapPropsToFields({
    filter: {
      search = '',
      begin,
      end,
      taskBeginId,
      taskEndId,
    } = {},
  }) {
    const { createFormField } = Form;

    return {
      search: createFormField({
        value: search,
      }),
      range: createFormField({
        value: begin && end ? [moment(begin), moment(end)] : [],
      }),
      taskBeginId: createFormField({
        value: taskBeginId,
      }),
      taskEndId: createFormField({
        value: taskEndId,
      }),
    };
  },
})(injectIntl(Filter));
