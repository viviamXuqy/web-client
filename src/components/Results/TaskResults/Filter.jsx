import React, { PureComponent } from 'react';
import { object, func } from 'prop-types';
import { Form, Button, Input, DatePicker, Select } from 'antd';
import { injectIntl } from 'react-intl';
import moment from 'moment';

import { TASK_TYPE } from '../../../constants/config';

class Filter extends PureComponent {
  static propTypes = {
    intl: object.isRequired,
    form: object.isRequired,
    onSearch: func.isRequired,
  }

  handleSubmit = e => {
    e.preventDefault();

    const { onSearch } = this.props;
    this.props.form.validateFieldsAndScroll((err, {
      range,
      ...values
    }) => {
      if (!err) {
        const [begin, end] = range;
        const options = {
          ...values,
          begin: new Date(begin).getTime(),
          end: new Date(end).getTime(),
        };

        onSearch(options);
      }
    });
  }

  render() {
    const { form, intl: { formatMessage } } = this.props;
    const { getFieldDecorator } = form;

    return (
      <div>
        <Form
          className="app-page__filter"
          layout="inline"
          onSubmit={this.handleSubmit}
        >
          <div className="fl">
            <Form.Item>
              {getFieldDecorator('search')(
                <Input className="result-input" placeholder={formatMessage({ id: 'App.form.rules.keyword' })} />,
              )}
            </Form.Item>
            <Form.Item>
              {getFieldDecorator('range')(
                <DatePicker.RangePicker
                  style={{ width: 330 }}
                  showTime={{
                defaultValue: [moment('00:00:00', 'HH:mm:ss'), moment('23:59:59', 'HH:mm:ss')],
              }}
                  format="YYYY-MM-DD HH:mm:ss"
                />,
              )}
            </Form.Item>
            <Form.Item label={formatMessage({ id: 'App.results.taskType' })}>
              {getFieldDecorator('type')(
                <Select>
                  <Select.Option value={TASK_TYPE.ALL}>{formatMessage({ id: 'App.results.all' })}</Select.Option>
                  <Select.Option value={TASK_TYPE.ANALYSIS}>{formatMessage({ id: 'App.results.1' })}</Select.Option>
                  <Select.Option value={TASK_TYPE.VIDEO}>{formatMessage({ id: 'App.results.2' })}</Select.Option>
                  <Select.Option value={TASK_TYPE.PIC}>{formatMessage({ id: 'App.results.3' })}</Select.Option>
                </Select>,
              )}
            </Form.Item>
            <Form.Item className="buttons-group">
              <Button type="primary" htmlType="submit" icon="search">
                {formatMessage({ id: 'App.control.search' })}
              </Button>
            </Form.Item>
          </div>
        </Form>
      </div>
    );
  }
}

export default Form.create({
  mapPropsToFields({
    filter: {
      search = '',
      begin,
      end,
      type = TASK_TYPE.ALL,
    },
  }) {
    const { createFormField } = Form;

    return {
      search: createFormField({
        value: search,
      }),
      range: createFormField({
        value: begin && end ? [moment(begin), moment(end)] : [],
      }),
      type: createFormField({
        value: type,
      }),
    };
  },
})(injectIntl(Filter));
