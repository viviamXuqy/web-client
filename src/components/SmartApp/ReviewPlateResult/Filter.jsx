import React, { PureComponent } from 'react';
import { object, func } from 'prop-types';
import { Form, Button, Input } from 'antd';
import { injectIntl } from 'react-intl';

import Cicon from '../../../shared/components/Cicon';

class Filter extends PureComponent {
  static propTypes = {
    intl: object.isRequired,
    form: object.isRequired,
    onSearch: func.isRequired,
  }

  handleSubmit = e => {
    e.preventDefault();

    const { onSearch } = this.props;

    this.props.form.validateFieldsAndScroll((err, {
      ...values
    }) => {
      if (!err) {
        const options = {
          ...values,
        };

        onSearch(options);
      }
    });
  }

  render() {
    const { form, intl: { formatMessage }, onSearch } = this.props;
    const { getFieldDecorator } = form;

    return (
      <div>
        <Form
          className="app-page__filter"
          layout="inline"
          onSubmit={this.handleSubmit}
        >
          <div className="fl">
            <Form.Item>
              {getFieldDecorator('plateNo')(
                <Input className="result-input" placeholder="请输入车牌号" />,
              )}
            </Form.Item>
            <Form.Item className="buttons-group">
              <Button type="primary" htmlType="submit" icon="search">
                {formatMessage({ id: 'App.control.search' })}
              </Button>
              <Button
                className="buttons-group"
                type="primary"
                onClick={onSearch}
              >
                <Cicon type="therefresh" />{formatMessage({ id: 'App.control.reset' })}
              </Button>
            </Form.Item>
          </div>
        </Form>
      </div>
    );
  }
}

export default Form.create({
  mapPropsToFields({
    filter: {
      plateNo = '',
    } = {},
  }) {
    const { createFormField } = Form;

    return {
      plateNo: createFormField({
        value: plateNo,
      }),
    };
  },
})(injectIntl(Filter));
