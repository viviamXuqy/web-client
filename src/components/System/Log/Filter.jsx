import React, { PureComponent } from 'react';
import { object, func } from 'prop-types';
import { Form, Button, DatePicker, Select } from 'antd';
import { injectIntl } from 'react-intl';
import moment from 'moment';

class Filter extends PureComponent {
  static propTypes = {
    intl: object.isRequired,
    form: object.isRequired,
    onSearch: func.isRequired,
  }

  handleSubmit = e => {
    e.preventDefault();

    const { onSearch } = this.props;

    this.props.form.validateFieldsAndScroll((err, {
      range,
      ...values
    }) => {
      if (!err) {
        const [begin, end] = range;
        const options = {
          ...values,
          begin: new Date(begin).getTime(),
          end: new Date(end).getTime(),
        };

        onSearch(options);
      }
    });
  }

  render() {
    const { form, intl: { formatMessage } } = this.props;
    const { getFieldDecorator } = form;

    return (
      <div>
        <Form
          className="app-page__filter"
          layout="inline"
          onSubmit={this.handleSubmit}
        >
          <div className="fl">
            <Form.Item label={formatMessage({ id: 'App.system.log.funType' })}>
              {getFieldDecorator('funType')(
                <Select style={{ width: 80 }}>
                  <Select.Option value={0}>{formatMessage({ id: 'App.system.log.funType0' })}</Select.Option>
                  <Select.Option value={1}>{formatMessage({ id: 'App.system.log.funType1' })}</Select.Option>
                  <Select.Option value={2}>{formatMessage({ id: 'App.system.log.funType2' })}</Select.Option>
                  <Select.Option value={3}>{formatMessage({ id: 'App.system.log.funType3' })}</Select.Option>
                </Select>,
              )}
            </Form.Item>
            <Form.Item label={formatMessage({ id: 'App.system.log.doType' })}>
              {getFieldDecorator('doType')(
                <Select>
                  <Select.Option value={0}>{formatMessage({ id: 'App.system.log.doType0' })}</Select.Option>
                  <Select.Option value={1}>{formatMessage({ id: 'App.system.log.doType1' })}</Select.Option>
                  <Select.Option value={2}>{formatMessage({ id: 'App.system.log.doType2' })}</Select.Option>
                  <Select.Option value={3}>{formatMessage({ id: 'App.system.log.doType3' })}</Select.Option>
                </Select>,
              )}
            </Form.Item>
            <Form.Item label={formatMessage({ id: 'App.system.log.time' })}>
              {getFieldDecorator('range')(
                <DatePicker.RangePicker
                  style={{ width: 330 }}
                  showTime={{
                defaultValue: [moment('00:00:00', 'HH:mm:ss'), moment('23:59:59', 'HH:mm:ss')],
              }}
                  format="YYYY-MM-DD HH:mm:ss"
                />,
              )}
            </Form.Item>
            <Form.Item className="buttons-group">
              <Button type="primary" htmlType="submit" icon="search">
                {formatMessage({ id: 'App.control.search' })}
              </Button>
            </Form.Item>
          </div>
        </Form>
      </div>
    );
  }
}

export default Form.create({
  mapPropsToFields({
    filter: {
      funType = 0,
      doType = 0,
      begin,
      end,
    },
  }) {
    const { createFormField } = Form;

    return {
      funType: createFormField({
        value: funType,
      }),
      doType: createFormField({
        value: doType,
      }),
      range: createFormField({
        value: begin && end ? [moment(begin), moment(end)] : [],
      }),
    };
  },
})(injectIntl(Filter));
