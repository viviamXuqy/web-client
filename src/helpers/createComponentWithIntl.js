import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line
import LocaleProvider from 'containers/Locale';
import { setStore } from '../utils/storage';
import { nodeWithIntlProp } from './intl-enzyme-test-helper';

const createComponentWithIntl = (children, props = { lang: 'zh-CN' }) => {
  setStore('locale', props.lang);
  return renderer.create(
    <LocaleProvider {...props}>
      {nodeWithIntlProp(children)}
    </LocaleProvider>,
  );
};

export default createComponentWithIntl;
