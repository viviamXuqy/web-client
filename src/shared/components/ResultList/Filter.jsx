import React, { PureComponent } from 'react';
import { object, func } from 'prop-types';
import { Form, Button, Input, DatePicker, message } from 'antd';
import { injectIntl } from 'react-intl';
import moment from 'moment';

class Filter extends PureComponent {
  static propTypes = {
    intl: object.isRequired,
    form: object.isRequired,
    onSearch: func.isRequired,
  }

  handleSubmit = e => {
    e.preventDefault();

    const { onSearch, intl } = this.props;

    this.props.form.validateFieldsAndScroll((err, {
      range,
      resultBeginId,
      resultEndId,
      ...values
    }) => {
      if (!err) {
        if ((resultBeginId && !resultEndId) || (resultEndId && !resultBeginId)) {
          message.warn(intl.formatMessage({ id: 'App.results.rules.resultId' }));
          return;
        }

        const [begin, end] = range;
        const options = {
          ...values,
          resultBeginId,
          resultEndId,
          begin: new Date(begin).getTime(),
          end: new Date(end).getTime(),
        };

        onSearch(options);
      }
    });
  }

  render() {
    const { form, intl: { formatMessage } } = this.props;
    const { getFieldDecorator } = form;

    return (
      <div>
        <Form
          className="app-page__filter2"
          layout="inline"
          onSubmit={this.handleSubmit}
        >
          <div className="fl">
            <Form.Item>
              {getFieldDecorator('search')(
                <Input className="result-input" placeholder={formatMessage({ id: 'App.form.rules.keyword' })} />,
              )}
            </Form.Item>
            <Form.Item>
              {getFieldDecorator('range')(
                <DatePicker.RangePicker
                  style={{ width: 330 }}
                  showTime={{
                defaultValue: [moment('00:00:00', 'HH:mm:ss'), moment('23:59:59', 'HH:mm:ss')],
              }}
                  format="YYYY-MM-DD HH:mm:ss"
                />,
              )}
            </Form.Item>
            <Form.Item>
              {getFieldDecorator('resultBeginId')(
                <Input className="result-input" placeholder={formatMessage({ id: 'App.results.beginResultId' })} />,
              )}
            </Form.Item>
            <div className="results-list2__lable">{formatMessage({ id: 'App.public.to' })}</div>
            <Form.Item>
              {getFieldDecorator('resultEndId')(
                <Input className="result-input" placeholder={formatMessage({ id: 'App.results.endResultId' })} />,
              )}
            </Form.Item>
            <Form.Item className="buttons-group">
              <Button type="primary" htmlType="submit" icon="search">
                {formatMessage({ id: 'App.control.search' })}
              </Button>
            </Form.Item>
          </div>
        </Form>
      </div>
    );
  }
}

export default Form.create({
  mapPropsToFields({
    filter: {
      search = '',
      begin,
      end,
      resultBeginId,
      resultEndId,
    },
  }) {
    const { createFormField } = Form;

    return {
      search: createFormField({
        value: search,
      }),
      range: createFormField({
        value: begin && end ? [moment(begin), moment(end)] : [],
      }),
      resultBeginId: createFormField({
        value: resultBeginId,
      }),
      resultEndId: createFormField({
        value: resultEndId,
      }),
    };
  },
})(injectIntl(Filter));
