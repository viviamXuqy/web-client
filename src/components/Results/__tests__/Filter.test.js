import React from 'react';
import createComponentWithIntl from 'helpers/createComponentWithIntl';
import { mountWithIntl } from 'helpers/intl-enzyme-test-helper';

import { message } from 'antd';

import Filter from 'components/Results/Filter';

const spyMessage = jest.spyOn(message, 'warn');

const onSearch = jest.fn();

const initProps = {
  onSearch,
};

const setup = (props = initProps) => {
  const component = <Filter {...props} />;
  const componentWithProvider = createComponentWithIntl(component);
  const wrapper = mountWithIntl(component);

  return {
    component,
    componentWithProvider,
    wrapper,
  };
};

describe('Filter component', () => {
  it('should render without errors', () => {
    const { wrapper } = setup();

    expect(wrapper).toBeTruthy();
  });

  it('should render correct things', () => {
    const { wrapper } = setup();

    expect(wrapper.find('.ant-form.ant-form-inline.app-page__filter').length).toBe(1);
  });

  it("should render right thing and don't change unexpected", () => {
    const { componentWithProvider } = setup();
    const tree = componentWithProvider.toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("should call 'onSearch' when form submit clicked", () => {
    const { wrapper } = setup();
    const { form } = wrapper.children().children().props();
    const formEl = wrapper.find('.ant-form');

    const filters = {
      search: 'xxx',
      begin: undefined,
      end: undefined,
      taskBeginId: '',
      taskEndId: '',
    };

    form.setFieldsValue(filters);

    formEl.simulate('submit');

    expect(onSearch).toBeCalledWith(filters);
  });

  it("should call 'message' when taskBeginId or taskEndId is undefined", () => {
    const { wrapper } = setup();
    const { form } = wrapper.children().children().props();
    const formEl = wrapper.find('.ant-form');

    const filters = {
      search: 'xxx',
      begin: undefined,
      end: undefined,
      taskBeginId: 1,
      taskEndId: '',
    };

    form.setFieldsValue(filters);

    formEl.simulate('submit');

    expect(spyMessage).toBeCalledWith('请输入任务ID');
  });
});
