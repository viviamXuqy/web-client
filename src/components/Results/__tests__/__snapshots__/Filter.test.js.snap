// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter component should render right thing and don't change unexpected 1`] = `
<div>
  <form
    className="ant-form ant-form-inline app-page__filter"
    onSubmit={[Function]}
  >
    <div
      className="fl"
    >
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <input
                className="ant-input result-input"
                data-__field={
                  Object {
                    "name": "search",
                    "value": "",
                  }
                }
                data-__meta={
                  Object {
                    "name": "search",
                    "originalProps": Object {
                      "className": "result-input",
                      "disabled": false,
                      "placeholder": "关键字",
                      "prefixCls": "ant-input",
                      "type": "text",
                    },
                    "ref": null,
                    "trigger": "onChange",
                    "validate": Array [],
                    "valuePropName": "value",
                  }
                }
                disabled={false}
                id="search"
                onChange={[Function]}
                onKeyDown={[Function]}
                placeholder="关键字"
                type="text"
                value=""
              />
            </span>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control has-success"
          >
            <span
              className="ant-form-item-children"
            >
              <span
                className="ant-calendar-picker"
                id="range"
                onBlur={[Function]}
                onFocus={[Function]}
                style={
                  Object {
                    "width": 330,
                  }
                }
                tabIndex={0}
              >
                <span
                  className="ant-calendar-picker-input ant-input"
                  onBlur={undefined}
                  onClick={[Function]}
                  onContextMenu={undefined}
                  onFocus={undefined}
                  onKeyDown={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={undefined}
                  onMouseLeave={undefined}
                  onTouchStart={[Function]}
                >
                  <input
                    className="ant-calendar-range-picker-input"
                    disabled={undefined}
                    placeholder="开始日期"
                    readOnly={true}
                    tabIndex={-1}
                    value=""
                  />
                  <span
                    className="ant-calendar-range-picker-separator"
                  >
                     ~ 
                  </span>
                  <input
                    className="ant-calendar-range-picker-input"
                    disabled={undefined}
                    placeholder="结束日期"
                    readOnly={true}
                    tabIndex={-1}
                    value=""
                  />
                  <span
                    className="ant-calendar-picker-icon"
                  />
                </span>
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <input
                className="ant-input result-input"
                data-__field={
                  Object {
                    "name": "taskBeginId",
                    "value": undefined,
                  }
                }
                data-__meta={
                  Object {
                    "name": "taskBeginId",
                    "originalProps": Object {
                      "className": "result-input",
                      "disabled": false,
                      "placeholder": "开始任务ID",
                      "prefixCls": "ant-input",
                      "type": "text",
                    },
                    "ref": null,
                    "trigger": "onChange",
                    "validate": Array [],
                    "valuePropName": "value",
                  }
                }
                disabled={false}
                id="taskBeginId"
                onChange={[Function]}
                onKeyDown={[Function]}
                placeholder="开始任务ID"
                type="text"
                value=""
              />
            </span>
          </div>
        </div>
      </div>
      <div
        className="results-list__lable"
      >
        至
      </div>
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <input
                className="ant-input result-input"
                data-__field={
                  Object {
                    "name": "taskEndId",
                    "value": undefined,
                  }
                }
                data-__meta={
                  Object {
                    "name": "taskEndId",
                    "originalProps": Object {
                      "className": "result-input",
                      "disabled": false,
                      "placeholder": "结束任务ID",
                      "prefixCls": "ant-input",
                      "type": "text",
                    },
                    "ref": null,
                    "trigger": "onChange",
                    "validate": Array [],
                    "valuePropName": "value",
                  }
                }
                disabled={false}
                id="taskEndId"
                onChange={[Function]}
                onKeyDown={[Function]}
                placeholder="结束任务ID"
                type="text"
                value=""
              />
            </span>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item buttons-group"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <button
                className="ant-btn ant-btn-primary"
                onClick={[Function]}
                type="submit"
              >
                <i
                  className="anticon anticon-search"
                />
                <span>
                  检索
                </span>
              </button>
            </span>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
